packages/cli/lang/en.lyaml

```diff
@@ -993,6 +1023,10 @@ en:
         logFeedbackMessage:
           feedbackHeader: "We'd love to hear your feedback!"
           feedbackMessage: "How are you liking the new projects and developer tools? \n > Run `{{#yellow}}hs feedback{{/yellow}}` to let us know what you think!\n"
+      publicApps:
+        errors:
+          noApps: "{{#bold}}No apps to convert{{/bold}}"
+          noAppsMessage: "The selected developer account {{#bold}}{{ accountName }}{{/bold}} doesn't have any apps that can be converted to the projects framework."
```

Comment:
Let's try to use the `uiAccountDescription` util as much as possible so that all of the account names in the logs are standardized

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/62039a6495b1b5eb20d8ddd746508b2daf69a8d4